src/N98/Magento/Command/Admin/User/LockCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace N98\Magento\Command\Admin\User;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+
+class LockCommand extends AbstractAdminUserCommand
+{
+    /**
+     * The number of days to lock for (by default)
+     * @var int
+     */
+    const LIFETIME_DEFAULT = 31;
+
+    /**
+     * Setup
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('admin:user:lock')
+            ->addArgument('username', InputArgument::REQUIRED, 'Admin username to lock')
+            ->addArgument('lifetime', InputArgument::OPTIONAL, 'Optional - lock lifetime (default one month)')
+            ->setDescription(
+                <<<HELP
+Enforce a lock on an admin user account. Specify the username and an optional lifetime parameter in seconds.
+HELP
+            );
+    }
+
+    /**
+     * @return bool
+     */
+    public function isEnabled()
+    {
+        return $this->getApplication()->isMagentoEnterprise();
+    }
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->detectMagento($output, true);
+        if (!$this->initMagento()) {
+            return;
+        }
+
+        $username = $input->getArgument('username');
+        $lifetime = $input->getArgument('lifetime') ?: $this->daysToSeconds(self::LIFETIME_DEFAULT);
+
+        $user = \Mage::getModel('admin/user')->loadByUsername($username);
+        if (!$user || !$user->getId()) {
+            $output->writeln("<error>Couldn't find admin username '{$username}'</error>");
+            return;
+        }
+
+        \Mage::getResourceModel('enterprise_pci/admin_user')->lock($user->getId(), 0, $lifetime);
```

Comment:
Ah, you're absolutely right. I've just overlooked it, it should prevent that effectively!